### DISCUSSION TITLE
How to get the latest fixes of PgWatch2?

### DISCUSSION THREAD

**Discussion by muhabbad** [cybertec-postgresql/pgwatch2#612] [Q&A]
Hello,

I want to get latest fixes of pgwach2 after installing the latest version from package. I dont't know how to to it and couldn't find any documentation clearly describe it. Can you help me? What should be the steps to get the latest fixes of pgwatch2 to apply existing installation? 

Thank you.

**Comment by pashagolub** [comment]
Hello. We have a chapter in out documentation about upgrading: https://pgwatch2.readthedocs.io/en/latest/upgrading.html
